specification/azurefleet/AzureFleet.Management/tspconfig.yaml

```diff
@@ -1,6 +1,6 @@
 parameters:
   "service-dir":
-    default: "sdk/azurefleet"
+    default: "sdk/computefleet"
```

Comment:
Shall we update this file to include JS emitter?  Otherwise, there will be no JS package in the SDK automation.
 cc @MaryGao